### PR TITLE
Add XXL size label

### PR DESCRIPTION
**Pull request by sarahbx**

### CODE REVIEW

**Comment by sarahbx** [comment]
@myakove can you trigger the CI please?

**Review by sarahbx — COMMENTED** [review]
this is a test

File: src/size_label_prs.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 def size_label_prs(data, pull):
     labels = get_labels(pull=pull)
     additions = data["pull_request"]["additions"]
-    label = None
+
+    comment = None
     if additions < 20:
         label = "Size/XS"
+        comment = "Thank you for a tiny PR!"
```
Comment (myakove) [inline src/size_label_prs.py:11]: Nice, but I prefer not to :)

File: src/size_label_prs.py [hunk 2]
```diff
@@ -20,10 +22,61 @@ def size_label_prs(data, pull):
...
         return
 
     else:
         print(f"Labeling {pull.title}: {label}")
         [pull.remove_from_labels(lb) for lb in labels if lb.lower().startswith("size/")]
         pull.add_to_labels(label)
+        print(f"Commenting on {pull.title}: {comment}")
+        if comment:
+            last_commit = list(pull.get_commits())[-1]
+            pull.create_review(commit=last_commit, body=comment)
```
Comment (myakove) [inline src/size_label_prs.py:39]: create_review will not comment last by default?

**Review by myakove — CHANGES_REQUESTED** [review]

File: src/size_label_prs.py [hunk 3]
```diff
@@ -20,10 +22,21 @@ def size_label_prs(data, pull):
     elif additions < 500:
         label = "Size/XL"
 
+    else:
+        label = "Size/XXL"
+        comment = "Please try to break up this PR, it is very large."
+
+    if not comment:
+        comment = label
```
Comment (myakove) [inline src/size_label_prs.py:30]: We need to add comment only if > 500

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by myakove** [comment]
please rebase.

File: src/size_label_prs.py (hunk 3 above)
Comment (sarahbx) [reply src/size_label_prs.py:30]: this was temporary, not intended to stay.

File: src/size_label_prs.py (hunk 2 above)
Comment (sarahbx) [reply src/size_label_prs.py:42]: I will re-verify. Was added during an 'actions' degradation

**Review by myakove — APPROVED** [review]

**Pull request merged**